enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/configuration/EnterpriseEditionSettings.java

```diff
@@ -32,7 +32,7 @@
 public class EnterpriseEditionSettings
 {
     @Description( "Specified names of id types (comma separated) that should be reused. " +
-                  "Currently only 'RELATIONSHIP' type is supported. " )
```

Comment:
Will this PR then enable `NODE` to be used? I'd expect not since it's merely adding tests.